Fix Communication Readme and Sample issues (#18977)
Azure/azure-sdk-for-net: sdk/communication/Azure.Communication.PhoneNumbers/README.md

### Get area codes

> Fetching area codes for geographic phone plans will require the location options queries set. You must include the chain of geographic locations traversing down the location options object returned by the GetLocationOptions API.

Reviewer: ```suggestion
Fetching area codes for geographic phone plans will require that the location options queries be set. You must include the chain of geographic locations which traverse down the location options returned by the `GetLocationOptions` API.
```
